crossing-training.lic

```diff
@@ -489,7 +498,9 @@ class CrossingTraining
     fput('release care') if data['cyclic'] && DRSpells.active_spells['Caress of the Sun']
     fput('release bes') if data['cyclic'] && DRSpells.active_spells['Bear Strength']
     fput('release fg') if data['cyclic'] && DRSpells.active_spells['Faenella\'s Grace']
-
+    fput('release rezz') if data['cyclic'] && DRSpells.active_spells['Ressurection']
```

Comment:
I think we should really refactor this section to release any active cyclics - instead of a bunch of special cases.